lorenzo/chain.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "lorenzo",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.lorenzo-protocol.xyz/",
+  "pretty_name": "Lorenzo Protocol",
+  "chain_type": "cosmos",
+  "chain_id": "lorenzo_8329-1",
+  "bech32_prefix": "lrz",
```

Comment:
Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
PR made, wait for merge. 

https://github.com/satoshilabs/slips/pull/1782